addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -151,4 +242,215 @@ options.registry.ImageTools.include({
+        this.styleSheet = sheetOwner.sheet;
+        return res;
+    },
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * Option method to set a css property in the mailing's custom stylesheet.
+     * Note: marks all the styles as important to make sure they take precedence
+     * on other stylesheets.
+     *
+     * @param {boolean|string} previewMode
+     * @param {string} widgetValue
+     * @param {Object} params
+     * @param {string} params.selectorText the css selector for which to apply
+     *                                     the css
+     * @param {string} params.cssProperty the name of the property to edit
+     *                                    (camel cased)
+     * @param {string} [params.toggle] if 'true', will remove the property if
+     *                                 its value is already the one it's being
+     *                                 set to
+     * @param {string} [params.activeValue] the value to set, if `widgetValue`
+     *                                      is not defined.
+     * @returns {Promise|undefined}
+     */
+    customizeCssProperty(previewMode, widgetValue, params) {
+        if (!params.selectorText || !params.cssProperty) {
+            return;
+        }
+        let value = widgetValue || params.activeValue;
+        if (params.cssProperty.includes('color')) {
+            value = weUtils.normalizeColor(value);
+        }
+        for (const selector of this._getSelectors(params.selectorText)) {
+            const rule = this._getRule(selector);
+            if (rule) {
+                // The rule exists: update it.
+                if (params.toggle === 'true' && rule.style.getPropertyValue(params.cssProperty) === value) {
+                    rule.style.removeProperty(params.cssProperty);
+                } else {
+                    // Convert the style to css text and add the new style (the
+                    // `style` property is readonly, we can only edit `cssText`).
+                    const cssTexts = [];
+                    for (const style of rule.style) {
+                        if (style !== params.cssProperty) {
+                            cssTexts.push(`${style}: ${rule.style[style]} !important;`);
+                        }
+                    }
+                    cssTexts.push(`${params.cssProperty}: ${value} !important;`);
+                    rule.style.cssText = cssTexts.join('\n'); // Apply the new css text.
+                }
+            } else {
+                // The rule doesn't exist: create it.
+                this.styleSheet.insertRule(`${selector} {
+                    ${params.cssProperty}: ${value} !important;
+                }`);
+            }
+        }
+        this._commitCss();
+    },
+    /**
+     * Option method to set a css property in the mailing's custom stylesheet.
```

Comment:
Looks like a copy paste of the previous method documentation.

Reply:
Oops